### PR TITLE
Test blob_kzg_commitments_merkle_proof with different blob counts

### PR DESCRIPTION
**Pull request by EvgeniyZh**
#### Test Improvements Proposal

**_run_blob_kzg_commitments_merkle_proof_test** - added an additional argument (blob_count) to enable testing multiple blobs and boundary value.  This test can be moved to a separate test module.

**test_get_custody_groups__multiple_seeds** -added as an example of replacing
*test_blob_kzg_commitments_merkle_proof__random_block_1* with random seeds. Some tests follow the same structure as *[test_compute_columns_for_custody_group__2](https://github.com/ethereum/consensus-specs/blob/2cc2cd94215abca26dde9afdf6f29ba214130e1a/tests/core/pyspec/eth2spec/test/fulu/networking/test_compute_columns_for_custody_group.py#L44)* 

**test_get_custody_groups__deterministic_heck** - added an extra check to ensure the value remains the same.

### CODE REVIEW

File: tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py [hunk 1]
```diff
@@ -74,3 +74,38 @@ def test_blob_kzg_commitments_merkle_proof__basic(spec, state):
...
+
+    for index, seed in enumerate([1111, 2222, 3333], start=1):
+        yield make_test(seed, index)
+
+
+@with_test_suite_name("BeaconBlockBody")
+@with_fulu_and_later
+@spec_state_test
+def test_blob_kzg_commitments_merkle_proof__multiple_blobs(spec, state):
+    max_blobs = spec.MAX_BLOBS_PER_BLOCK_FULU / 2
```
[the remark this answers is not in the document]
Comment (jtraglia) [reply tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py:100]: Also, nit: let's use `//` for integer division.

[the remark this answers is not in the document]
Comment (jtraglia) [reply tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py:100]: Also again, this needs to be `spec.config.MAX_BLOBS_PER_BLOCK_FULU`.

[the remark this answers is not in the document]
Comment (EvgeniyZh) [reply tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py]: Thanks for the feedback. Updated ,renamed to *blob_count*

File: tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py [hunk 2]
```diff
@@ -74,3 +74,38 @@ def test_blob_kzg_commitments_merkle_proof__basic(spec, state):
...
+        def inner_test():
+            rng = random.Random(seed)
+            try:
+                yield from _run_blob_kzg_commitments_merkle_proof_test(spec, state, rng=rng)
+            except Exception as e:
+                raise AssertionError(f"Failed on: {seed} index: {index}, {e}")
+        return inner_test
+
+    for index, seed in enumerate([1111, 2222, 3333], start=1):
+        yield make_test(seed, index)
```
[the remark this answers is not in the document]
Comment (EvgeniyZh) [reply tests/core/pyspec/eth2spec/test/fulu/merkle_proof/test_single_merkle_proof.py]: Done, test is removed.

File: tests/core/pyspec/eth2spec/test/fulu/networking/test_get_custody_groups.py [hunk 3]
```diff
@@ -104,3 +104,33 @@ def test_get_custody_groups__2(spec):
...
+        def inner_test():
+            rng = random.Random(seed)
+            try:
+                yield from _run_get_custody_groups(spec, rng)
+            except Exception as e:
+                raise AssertionError(f"Failed on: {seed} index: {index}, {e}")
+        return inner_test
+
+    for index, seed in enumerate([1111, 2222, 3333], start=1):
+        yield make_test(seed, index)
```
[the remark this answers is not in the document]
Comment (EvgeniyZh) [reply tests/core/pyspec/eth2spec/test/fulu/networking/test_get_custody_groups.py]: Done.
